pandas/core/generic.py

```diff
@@ -9589,16 +9685,16 @@ def describe(
 
         Excluding numeric columns from a ``DataFrame`` description.
 
-        >>> df.describe(exclude=[np.number])
+        >>> df.describe(exclude=[np.number]) # doctest: +SKIP
```

Comment:
How come you are adding doctest skips opposed to our current pytest -k approach? Think we should be consistent.

Reply:
The problem with describe, is that the output can be random, If you know how to skip a specific line in the output, it would be great!